### DISCUSSION TITLE
[Enhancement] Make Maps independent from Google and Bing Maps

### DISCUSSION THREAD

**Discussion by trevelyanuk** [dotnet/maui#150] [General]
## Summary
Looking at the roadmap, currently there appears to be no indication that Maps would allow for a custom tile source to be used. Admittedly, it doesn't specify any dependency on Google, Apple or Bing Maps, but I am basing this suggestion on the assumption that Maps in MAUI would build on existing Xamarin Forms implementations.

## API Changes
It would be useful to be able to specify a tile source, or a provider, for a Map, whereby the developer can choose the tile source. Having a platform default would make sense, facilitating API keys for certain providers makes sense too; but we shouldn't be tied to using a specific provider or having to look for other solutions to give us that flexibility, especially when Google, for example, charge for access to their API (currently).

## Intended Use Case
* Any map solution where coupling to Google (or any other map provider) is both unfeasible or not desired.
* Any map solution that does not, or aims to avoid, Google Play services (especially if they aren't distributing the application through Google's services)
* Any map solution that is offline or operates within an organisation's network only

**Comment by jingliancui** [comment]
Awesome idea !  Google play services is not permitted in China. Specially on android side we must use the 3rd party maps library made with the companies in China. I thinks it's good to export some interfaces like the `Renderer` feature to embed other implements of the map libraries.
